MainTel/MainTel-Non-standard-Operations/MTCS-Non-standard-Procedures/TMA-Non-standard-Procedures/TMA-Restart-MTMount-Operation-Manager-(OM).rst

```diff
@@ -0,0 +1,154 @@
+
+
+.. prompt:: python
+
+ mtmount-operation-manager.service - LLST MTMount Operation Manager service
+   Loaded: loaded (/usr/lib/systemd/system/mtmount-operation-manager.service; enabled; vendor preset: disabled)
+   Active: active (running) since Fri 2024-03-15 08:30:17 UTC; 5 days ago
+ Main PID: 1036 (mtmount-operati)
+    Tasks: 20
+   Memory: 65.8M
+   CGroup: /system.slice/mtmount-operation-manager.service
+           └─1036 /usr/bin/mtmount-operation-manager
+ 
+ Mar 15 08:30:17 lsst systemd[1]: Started LLST MTMount Operation Manager service.
+ Hint: Some lines were ellipsized, use -l to show in full.
+
+Note that in this case, the status is "active (running) since Fri 2024-03-15 08:30:17 UTC; 5 days ago".
+
+4. **Stop MTMount Operation Manager**: To stop the service, replace "status" with "stop" in the previous command:
+
+.. prompt:: python
+
+ [lsst@lsst ~]$ sudo systemctl stop mtmount-operation-manager.service
+
+
+5. **Start MTMount Operation Manager**: Wait for one to two minutes before starting the operation manager again:
+
+.. prompt:: python
+
+ [lsst@lsst ~]$ sudo systemctl start mtmount-operation-manager.service
+
+
+Wait for up to 5 minutes before proceeding to the next step and check that the OM status is:
+
+    Active: active (running) since ...
+
+6. **Open the EUI Application**: Open the EUI application to resume operations. If you don't know how to open it, refer to the "Opening TMA EUI" section in `this document <https://confluence.lsstcorp.org/display/OOD/How+create+a+VNC+connection+to+TMA+EUI>`__.
+
+
+
+Post-Condition
+==============
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+Upon completion of the procedure outlined above, the following post-conditions are expected:
+
+.. _Title-of-Troubleshooting-Procedure-Critical-Step-2:
+
+1. The MTMount Operation Manager service is running again.
+2. The TMA EUI application is operational and MTMount can be enabled without errors.
+
+Ensuring these post-conditions confirms the successful completion of the OM restart procedure and guarantees the continuity of telescope operations.
+
+.. _Title-of-Troubleshooting-Procedure-Contingency:
+
+Contingency
+===========
+
+If the procedure was not successful, report the issue in [relevant Slack channel] and/or activate the :ref:`Out of hours support <Safety-out-of-hours-support>`.
```

Comment:
Replace **relevant Slack channel** for the #summit-simonyi.

Reply:
haha I missed this!, my bad, thanks